browser/browsing_data/brave_clear_browsing_data.h

```diff
@@ -0,0 +1,45 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_BROWSING_DATA_BRAVE_CLEAR_BROWSING_DATA_H_
+#define BRAVE_BROWSER_BROWSING_DATA_BRAVE_CLEAR_BROWSING_DATA_H_
+
+namespace content {
+class BrowsingDataRemover;
+}
+class Profile;
+class BraveClearDataOnExitTest;
+
+namespace content {
+
+class BraveClearBrowsingData {
+ public:
+  // Clears browsing data for all loaded non-off-the-record profiles.
+  // Profile's *OnExit preferences determine what gets cleared.
+  static void ClearOnExit();
```

Comment:
Maybe add a comment regarding the blocking nature of this method

Reply:
Done.